diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/StatelessClassesRuleWarnTest.kt

```diff
@@ -0,0 +1,83 @@
+package org.cqfn.diktat.ruleset.chapter6
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames.OBJECT_IS_PREFERRED
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.classes.StatelessClassesRule
+import org.cqfn.diktat.util.LintTestBase
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class StatelessClassesRuleWarnTest: LintTestBase(::StatelessClassesRule) {
+    private val ruleId = "$DIKTAT_RULE_SET_ID:stateless-class"
+
+    @Test
+    @Tag(OBJECT_IS_PREFERRED)
+    fun `should not trigger on class not extending any interface`() {
```

Comment:
Oh, I thought right now that we can declare properties in the interface.
Will it be ok to convert it to the object?